src/SDK/Sdk.php

```diff
@@ -22,6 +23,7 @@ public function __construct(
         private readonly LoggerProviderInterface $loggerProvider,
         private readonly EventLoggerProviderInterface $eventLoggerProvider,
         private readonly TextMapPropagatorInterface $propagator,
+        private readonly HookManager $hookManager,
```

Comment:
This might just be a temporary home for hook manager, I'm not convinced that it belongs in the SDK.

Reply:
removed